Sources/WebSocket.swift

```diff
@@ -531,7 +531,7 @@ open class WebSocket : NSObject, StreamDelegate, WebSocketClient, WSStreamDelega
         request.setValue("\(url.host!):\(port!)", forHTTPHeaderField: headerWSHostName)
 
         var path = url.absoluteString
-        let offset = (url.scheme?.characters.count ?? 2) + 3
+        let offset = (url.scheme?.count ?? 2) + 3
```

Comment:
This change will break for anyone not yet on Xcode 9. Might be best to do the following instead:
```
#if swift(>=3.2)
let offset = (url.scheme?.count ?? 2) + 3
#else
let offset = (url.scheme?.characters.count ?? 2) + 3
#endif
```

Reply:
Nevermind just saw we're already Swift 4 🙂

Reply:
This deprecation message actually is only in Xcode 9.1 but let's get ready for it. ;)